src/pool.ts

```diff
@@ -1,22 +1,53 @@
+import {
+  BigInt,
+  BigDecimal
+} from '@graphprotocol/graph-ts'
 import {
   Staked,
   Withdrawn,
   RewardAdded,
   RewardPaid
 } from '../generated/templates/Pool/Pool'
+import {
+  Pool,
+  Token
+} from '../generated/schema'
+import {
+  exponentToBigDecimal
+} from './helpers'
+
+const EIGHTEEN_BI = BigInt.fromI32(18)
```

Comment:
maybe we can rename this to EIGHTEEN_BN if we are trying to make reference to a big number

Reply:
There are two types of big numbers in subgraphs; big decimals and big integers. There are some constants in the code base that exist for both (e.g. `ZERO_BD` and `ZERO_BI`) so I just reused that naming scheme for consistency